client/src/App.js

```diff
@@ -1,11 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
 import Register from './components/Register';
+import Login from './components/Login';
+import Home from './components/Home';
+import {Switch, Route} from 'react-router-dom';
 
 function App() {
   return (
     <div className="App">
-      <Register />
+      <Switch>
+        <Route exact path='/' component={Home}/>
+        <Route path='/login' component={Login}/>
+        <Route path='/register' component={Register}/>
+      </Switch>
```

Comment:
nit: You can wrap the <BrowserRouter> here instead of in index.js so we keep everything contained in App.js